src/Simulation/Simulators/QuantumSimulator/ClassicalControl.cs

```diff
@@ -0,0 +1,288 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Microsoft.Quantum.Simulation.Core;
+
+namespace Microsoft.Quantum.Simulation.Simulators
+{
+    public partial class QuantumSimulator
+    {
+        #region Static Methods
+
+        private static void RunClause(ICallable op, OperationFunctor type, IQArray<Qubit>? ctrls)
```

Comment:
I was looking around and it seems like most things in QuantumSimulator are missing doc comments, but they would be nice to have for new code. :)

Reply:
done

Reply:
Thanks!